microsoft-edge/webview2/concepts/process-model.md

```diff
@@ -19,21 +19,23 @@ The WebView2 Runtime uses the same process model as the Microsoft Edge browser.
 <!-- ====================================================================== -->
 ## Processes in the WebView2 Runtime
 
-A collection of WebView2 Runtime processes, or WebView2 Process Group, is composed of the following processes:
+A collection of WebView2 Runtime processes is called a _WebView2 process group_.  A WebView2 process group is composed of the following processes:
 *  A single browser process.
-*  One or more render processes.
+*  One or more renderer processes.
 *  Other helper processes, such as the GPU process and the Audio service process.
 
 :::image type="complex" source="../media/process-model-1.png" alt-text="Process 1" lightbox="../media/process-model-1.png":::
    Process 1
 :::image-end:::
 
-Other than the browser process, the number and presence of processes in this collection can change as a WebView2 application makes use of WebView2 features.  For example, creating a new WebView from the same `CoreWebView2Environment` with a different domain in the `Source` property will usually start a new renderer process.  The logic that controls when these extra processes are created depends on the Chromium architecture, and is beyond the scope of the WebView2 Runtime.
+The number and presence of processes in a WebView2 process group can change as a WebView2 application makes use of WebView2 features.  (However, there's only a single, specific browser process in a WebView2 process group.)  For example, creating a new WebView from the same `CoreWebView2Environment`, but with a different domain in the `Source` property, will usually start a new renderer process.
 
 For example, the number of renderer processes varies based on the following conditions:
```

Comment:
I think "For example" can be removed. This should also be "could vary" instead of "varies", the actual behavior really depends on the browser defaults and feature overrides, though that's not necessarily something to put here, which is the reason I added the note that it depends on the Chromium architecture.

Reply:
thanks.  typing this comment from the .  web-based VS Code.  I'm making the changes.

notes: The Style Guide, in the "can" page, appears to say to avoid the word 'could', though the full story is more complex and you have to wonder what context is assumed, at https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may - "Don't substitute _could_ for _can_ unless you're referring to the past."